staging/src/github.com/kubeedge/api/README.md

```diff
@@ -0,0 +1,10 @@
+# API
+Schema of the external API types that are served by KubeEdge.
+
+# Purpose
+This library is the canonical location of the KubeEdge API definition.It will be published separately to avoid diamond dependency problems for users who depend on multiple KubeEdge components.
```

Comment:
```suggestion
This library is the canonical location of the KubeEdge API definition. It will be published separately to avoid diamond dependency problems for users who depend on multiple KubeEdge components.
```

Reply:
Done